src/Umbraco.Core/Persistence/Mappers/DomainMapper.cs

```diff
@@ -19,6 +19,7 @@ protected override void DefineMaps()
             DefineMap<UmbracoDomain, DomainDto>(nameof(UmbracoDomain.RootContentId), nameof(DomainDto.RootStructureId));
             DefineMap<UmbracoDomain, DomainDto>(nameof(UmbracoDomain.LanguageId), nameof(DomainDto.DefaultLanguage));
             DefineMap<UmbracoDomain, DomainDto>(nameof(UmbracoDomain.DomainName), nameof(DomainDto.DomainName));
+            DefineMap<UmbracoDomain, DomainDto>(nameof(UmbracoDomain.SortOrder), nameof(DomainDto.SortOrder));
```

Comment:
Seems like a mapper can be used instead of the factory methods, but I can't find documentation about the differences (most factories in `Umbraco.Core.Persistence.Factories` also have a mapper in `Umbraco.Core.Persistence.Mappers`).